setup.cfg

```diff
@@ -1,5 +1,6 @@
 [flake8]
 exclude=build,.git,.ve*,.hypothesis
-ignore=E123,E128,E265,E501,W601,W293
+# extend-ignore uses flake8's default ignore list, and adds to it
+extend-ignore=E128,E265,E501,W601,W293,W605
```

Comment:
Can we document why we are adding exceptions in https://opendataservices.gitbook.io/developer-docs/code/code-python#coding-style ? That will help share knowledge round the team. Thanks :-)